examples/preact/basic-file-based/src/routes/_pathlessLayout/_nested-layout.tsx

```
@@ -0,0 +1,35 @@
+import { createFileRoute } from '@tanstack/preact-router'
```

ESLint

[error] '@tanstack/preact-router' imported multiple times.

(import/no-duplicates)